csrc/serde/fusion_cache_generated.h

```diff
@@ -5,6 +5,13 @@
 
 #include "flatbuffers/flatbuffers.h"
 
+// Ensure the included flatbuffers.h is the same version as when this file was
+// generated, otherwise it may not be compatible.
+static_assert(
+    FLATBUFFERS_VERSION_MAJOR == 2 && FLATBUFFERS_VERSION_MINOR == 0 &&
+        FLATBUFFERS_VERSION_REVISION == 7,
+    "Non-compatible flatbuffers version included");
+
```

Comment:
After this change my local PyTorch build fails because `#include "flatbuffers/flatbuffers.h"` resolves to the Pytorch submodule which right now in the main branch is 2.0.5: https://github.com/google/flatbuffers/blob/d0cede9c90c5257537c293517a21376408b549fa/include/flatbuffers/base.h#L143

Thoughts?

Reply:
Did you update the flatbuffer submodule?
```
git submodule sync
git submodule update --init --recursive
```

Reply:
I linked above to the file following the main branch, do you expect anything different?

Reply:
Oh that may be our discrepancy: I am building PyTorch from source and pointing a Nvfuser: seems like nvfuser is diverging here?

Reply:
@zasdfgbnm I created a PR #285 to revert to the old flatbuffer.